doc/Installation/Installation-Ubuntu-1604-Nginx.md

```diff
@@ -9,9 +9,11 @@ source: Installation/Installation-Ubuntu-1604-Nginx.md
 ```bash
 apt-get install mariadb-server mariadb-client
 systemctl restart mysql
-mysql -uroot -p
+mysql -u root -p
```

Comment:
Not sure why this has been changed as -uroot is a valid way to specify the username